charts/karpenter-crd/templates/karpenter.sh_nodeclaims.yaml

```diff
@@ -35,6 +35,10 @@ spec:
         - jsonPath: .metadata.creationTimestamp
           name: Age
           type: date
+        - jsonPath: .status.imageID
+          name: ImageID
```

Comment:
interesting point surrounding the convo of image naming. 

I think this aligns ok with our chosen naming.

Reply:
Yes. Note that this is not a new field, just added to wide printing. It is populated at the end of the Create from VM:
https://github.com/Azure/karpenter-provider-azure/blob/0f1aca1bb5d16553421cf0d7c42697fad4848ed9/pkg/cloudprovider/cloudprovider.go#L361